nav2_gps_waypoint_follower_demo/params/city_world_gps_waypoints.yaml

```diff
@@ -0,0 +1,26 @@
+gps_waypoint_follower_demo:
```

Comment:
Should this package not also include the map / gazebo model you were using? That way these waypoints are meaningful relative to some simulation environment.

Reply:
I can add them but since the waypoints were acquired somewhere close to center of Gazebo simulation world, i think this waypoints will make sense quite for a lot of cases. Also the world I use have various gazebo models in it so including them here might be not that relevant to the purpose.

Reply:
Well the value of the demo / tutorial is so that people can get up and running to test things fast (hopefully with an accompanying tutorial), so including the entire environment that these GPS waypoints were designed for would be good. If you just used models from the gazebo farm (e.g. not your own custom meshes), then you should only have a world file right?

Reply:
I have added all gazebo models and the world I used to collect GPS sample points inside but since there is no GPS attached to anything (there is no robot), I am not sure how useful that would be.